src/main/java/net/coderbot/iris/pipeline/ShadowRenderer.java

```diff
@@ -292,7 +292,7 @@ public void renderShadows(WorldRendererAccessor worldRenderer, Camera playerCame
 
 		// TODO: Support shadow clear color directives & disable buffer clearing
 		// Ensure that the color and depth values are cleared appropriately
-		RenderSystem.clearColor(0.0f, 0.0f, 0.0f, 0.0f);
+		RenderSystem.clearColor(255, 255, 255, 1);
```

Comment:
Color values should be in the range of 0.0F to 1.0F when calling clearColor, not 0 to 255.

Reply:
Fixed.